CHANGELOG

```diff
@@ -11,6 +11,7 @@ about future releases, check `milestones`_ and :doc:`/about/vision`.
 - Bug #41 - demo/accounts.py cannot be completed automatically: user
   interaction is required to activate account. Added an interactive prompt
   in demo. Removed demo/accounts.py from tests.
+- Feater #34 - add timeout option
```

Comment:
Note: I do not expect the PR to provide CHANGELOG. I mean, it's ok to provide the CHANGELOG, but it is not blocker. I think CHANGELOG review and update is part of maintainer's job, i.e. I take care of the CHANGELOG on the merge commit.
See also http://tech.novapost.fr/changelog-howto-en.html#maintain-changelog-as-you-merge-pull-requests